kitsune/sumo/static/sumo/js/form-wizard.js

```diff
@@ -34,18 +35,31 @@ export class FormWizard extends HTMLElement {
   static get markup() {
     return `
       <template>
-        <div class="form-wizard-content">
-          <h2>${gettext("Wizard header")}</h2>
-          <section>
-            <ul id="step-indicator"></ul>
-            <slot name="active"></slot>
-          </section>
+        <div class="form-wizard-root">
+          <div class="form-wizard-content">
+            <h2>${gettext("Wizard header")}</h2>
+            <section>
+              <ul id="step-indicator"></ul>
+              <slot name="active"></slot>
+            </section>
+          </div>
+          <progress max="100" value="0"></progress>
+          <div class="form-wizard-disqualified">
+            <div class="disqualification" reason="need-fx-desktop">
+              <span class="disqualification-header"><img class="warning-icon" src="${warningImageURL}" aria-hidden="true"/>${gettext("Use Firefox to continue")}</span>
+              <div class="disqualification-message">${gettext("To use the setup assistant or access the settings for backing up your Firefox data, please visit this page using Firefox on a desktop device.")}</div>
+            </div>
+
+            <div class="disqualification" reason="uitour-broken">
+              <div class="disqualification-message">${gettext("The setup assistant is currently unavailable for your version of Firefox. However, you can still perform a manual backup of your data by following the steps outlined in")} <a href="/kb/profile-manager-create-remote-switch-firefox-profiles">${gettext("this article.")}</a></div>
```

Comment:
Please never do this.
https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#avoid-concatenations-use-placeholders-instead

These two strings landed as separate strings with zero context, I assume concatenation only because `this article.` stars with lowercase and ends with a period.

This is effectively impossible to localize if a language requires "this article" in the middle of the other sentence.

Reply:
Thanks, @flodolo. What is the correct way to localize this in the JavaScript layer of this Django application?

Reply:
@flodolo 👍 That makes sense. I'm at fault for recommending this approach.

Reply:
hey @flodolo let me know if this approach using `interpolate` helps address the issue: https://github.com/mozilla/kitsune/pull/5517

if not I'm open to other suggestions!